composeApp/src/androidInstrumentedTest/kotlin/org/ooni/probe/test/OnboardingTest.kt

```diff
@@ -0,0 +1,63 @@
+package org.ooni.probe.test
+
+import androidx.compose.ui.test.isDisplayed
+import androidx.compose.ui.test.junit4.createEmptyComposeRule
+import androidx.compose.ui.test.onNodeWithContentDescription
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performClick
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.test.runTest
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.ooni.probe.data.models.SettingsKey
+import org.ooni.probe.test.helpers.CleanTestRule
+import org.ooni.probe.test.helpers.preferences
+import org.ooni.probe.test.helpers.start
+
+@RunWith(AndroidJUnit4::class)
+class OnboardingTest {
+    @get:Rule
+    val clean = CleanTestRule()
+
+    @get:Rule
+    val compose = createEmptyComposeRule()
+
+    @Before
+    fun setUp() {
+        start()
+    }
+
+    @Test
+    fun onboarding() =
+        runTest {
+            compose.waitUntil { compose.onNodeWithText("What is OONI Probe?").isDisplayed() }
+            compose.onNodeWithText("Got It").performClick()
```

Comment:
Would this not be better to run these with Tag/Id so we can Once in a while run the tests with different device configuration for language like RTL and others

Reply:
That's a good point. 

In one hand, testing the actual strings as the users see them, will make sure they are seeing the right content we want. And it's simpler, because we don't have to add test tags to everything. But on the other hand, to test in other languages, we need to write the tests for those languages, with the changed strings.

If we use test tags everywhere, the tests would still pass if we switch languages and text direction. But we wouldn't be testing if the users are actually seeing the right text they should be seeing.

There are upsides and downsides to both methods. What would you feel is the best approach?